src/app/cube/content-pages/editorial-process/editorial-process.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, NgModule, OnInit } from '@angular/core';
+import { sections } from './copy';
+
+@Component({
+  selector: 'clark-editorial-process',
+  templateUrl: './editorial-process.component.html',
+  styleUrls: ['./editorial-process.component.scss']
+})
+export class EditorialProcessComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+  title = 'Editorial Process';
+  copy = sections;
+
+  get tabs(){
```

Comment:
What does this do?

Reply:
The way Katelyn explained it to me, is that the copy.ts file can only "talk to" the component.ts file, and the HTML file can only "talk to" the component.ts file. So that is the way to connect the copy.ts and the HTML file.

Reply:
You can also load this in as a variable of the component in the constructor on creation of the component or in ngOnit when the component is ready to render.